backend/tests/controllers/volunteerController.test.js

```
@@ -0,0 +1,124 @@
+jest.mock('../../src/services/volunteerService');
+jest.mock('../../src/utils/responseHelper');
+
+beforeAll(() => {
+  jest.spyOn(console, 'error').mockImplementation(() => {});
+});
+afterAll(() => {
+  console.error.mockRestore();
+});
+
+describe('volunteerController', () => {
+  let req, res;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)